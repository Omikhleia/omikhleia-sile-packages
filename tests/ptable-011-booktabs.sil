\begin[papersize=a6]{document}
\nofolios
\script[src=packages/ptable]
% Table with cell border and padding configuration at cell level:
% This table example is in the LaTeX booktabs package, let us try to reproduce it.

\center{\parbox[width=80%fw, strut=character]{%
\begin[cols=34%fw 33%fw 33%fw, cellborder=0]{ptable}
  \begin{row}
    \cell[span=2,border=0.8pt 0.4pt 0 0]{\center{Item}}
    \cell[border=0.8pt 0 0 0]{}
  \end{row}
  \begin{row}
    \cell[border=0 0.8pt 0 0, padding=4pt 4pt 0 4pt]{\noindent{}Animal}
    \cell[border=0 0.8pt 0 0]{Description}
    \cell[border=0 0.8pt 0 0, padding=4pt 4pt 4pt 0]{\raggedleft{Price ($)}}
  \end{row}
  \begin{row}
    \cell[valign=top, padding=4pt 1pt 0 4pt]{\noindent{}Gnat}
    \celltable[span=2]{
      \row{\cell[padding=4pt 1pt 4pt 4pt]{per gram}\cell[padding=4pt 1pt 4pt 0]{\raggedleft{13.65}}}
      \row{\cell[padding=1pt 1pt 4pt 4pt]{each}\cell[padding=1pt 1pt 4pt 0]{\raggedleft{0.01}}}
    }
  \end{row}
  \begin{row}
    \cell[padding=2pt 1pt 0 4pt]{\noindent{}Gnu}
    \cell[padding=2pt 1pt 4pt 4pt]{stuffed}
    \cell[padding=2pt 1pt 4pt 0]{\raggedleft{92.50}}
  \end{row}
  \begin{row}
    \cell[padding=2pt 1pt 0 4pt]{\noindent{}Emu}
    \cell[padding=2pt 1pt 4pt 4pt]{stuffed}
    \cell[padding=2pt 1pt 4pt 0]{\raggedleft{33.33}}
  \end{row}
  \begin{row}
    \cell[border=0 0.8pt 0 0, padding=2pt 4pt 0 4pt]{\noindent{}Armadillo}
    \cell[border=0 0.8pt 0 0, padding=2pt 4pt 4pt 4pt]{frozen}
    \cell[border=0 0.8pt 0 0, padding=2pt 4pt 4pt 0]{\raggedleft{8.99}}
  \end{row}
\end{ptable}
}}

\end{document}