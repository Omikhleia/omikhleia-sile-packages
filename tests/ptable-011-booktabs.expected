Set paper size 	595.275597	841.8897729
Begin page
Mx 	190.4882
My 	60.2488
Draw line	190.4882	48.0945	143.5805	0.8000
Draw line	190.4882	66.6902	143.5805	0.4000
Mx 	252.7423
Set font 	Gentium Plus;10;400;;normal;;LTR
T	44 87 72 80 w=19.0723	(Item)
Draw line	334.0687	48.0945	70.7187	0.8000
Mx 	190.4882
My 	78.8445
Draw line	190.4882	85.2859	72.8617	0.8000
T	36 81 76 80 68 79 w=29.5117	(Animal)
Draw line	263.3499	85.2859	70.7187	0.8000
Mx 	287.3499
T	39 72 86 70 85 76 83 87 76 82 81 w=47.4463	(Description)
Draw line	334.0687	85.2859	70.7187	0.8000
Mx 	370.2259
T	51 85 76 70 72 w=20.8887	(Price)
Mx 	393.7571
T	11 w=3.1689	(()
Mx 	396.9261
T	7 w=4.6924	($)
Mx 	401.6185
T	12 w=3.1689	())
Mx 	190.4882
My 	102.2380
My 	97.4402
T	42 81 68 87 w=19.3799	(Gnat)
Mx 	263.3499
My 	95.9402
Mx 	287.3499
My 	97.4402
T	83 72 85 w=13.8770	(per)
Mx 	303.8700
T	74 85 68 80 w=21.4600	(gram)
Mx 	383.7278
T	20 22 17 25 24 w=21.0596	(13.65)
Mx 	263.3499
My 	110.0359
Mx 	287.3499
T	72 68 70 75 w=19.0967	(each)
Mx 	388.4202
T	19 17 19 20 w=16.3672	(0.01)
Mx 	190.4882
My 	123.1316
My 	123.6316
T	42 81 88 w=16.6455	(Gnu)
Mx 	287.3499
T	86 87 88 747 72 71 w=28.6426	(stuffed)
Mx 	383.7278
T	28 21 17 24 19 w=21.0596	(92.50)
Mx 	190.4882
My 	136.7273
My 	137.2273
T	40 80 88 w=18.2910	(Emu)
Mx 	287.3499
T	86 87 88 747 72 71 w=28.6426	(stuffed)
Mx 	383.7278
T	22 22 17 22 22 w=21.0596	(33.33)
Mx 	190.4882
My 	151.8230
Draw line	190.4882	157.2644	72.8617	0.8000
My 	150.8230
T	36 85 80 68 71 76 79 79 82 w=40.8936	(Armadillo)
Draw line	263.3499	157.2644	70.7187	0.8000
Mx 	287.3499
T	73 85 82 93 72 81 w=26.4795	(frozen)
Draw line	334.0687	157.2644	70.7187	0.8000
Mx 	388.4202
T	27 17 28 28 w=16.3672	(8.99)
End page
Finish
